tests_doc.gemspec

```diff
@@ -13,12 +13,12 @@ Gem::Specification.new do |spec|
   spec.homepage      = "https://github.com/taskrabbit/tests_doc"
   spec.license       = "MIT"
 
-  spec.files         = `git ls-files`.split($/)
+  spec.files         = `git ls-files`.split($/).reject { |f| f =~ /^examples/ }
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "rails", '>= 4'
+  spec.add_dependency "rails", '~> 4.0'
```

Comment:
Let's leave the greater than? Maybe add `< 6.0`

Reply:
okay, i'll do it.

Reply:
@kbacha it's done now.